File: app/src/main/java/com/mlieou/yaara/model/BitTorrent.java
package com.mlieou.yaara.model;

import java.util.List;

/**
 * Created by mlieou on 1/4/18.
 */

public class BitTorrent {

    private List<List<String>> announceList;
    private String comment;
    private long creationDate;
    private String mode;
    private BitTorrentInfo info;

    public String getComment() {
        return comment;
    }

    public long getCreationDate() {
        return creationDate;
    }

    public List<List<String>> getAnnounceList() {
        return announceList;
    }

    public String getMode() {
        return mode;
    }

    public BitTorrentInfo getInfo() {
        return info;
    }

    public class BitTorrentInfo {
        public String name;
    }

}

File: app/src/main/java/com/mlieou/yaara/model/BitTorrent.kt
package com.mlieou.yaara.model

/**
 * Created by mlieou on 5/19/18.
 */

class BitTorrent(val announceList : List<List<String>>,
                 val comment : String,
                 val creationDate : Long,
                 val mode : String,
                 val bitTorrentInfo: BitTorrentInfo)

class BitTorrentInfo(val name: String)